rts/lra/lra-proxy/api/pom.xml

```diff
@@ -42,12 +42,14 @@
             <scope>provided</scope>
         </dependency>
         <!-- JAXRS 2 Client API -->
+<!--
         <dependency>
             <groupId>org.jboss.spec.javax.ws.rs</groupId>
-            <artifactId>jboss-jaxrs-api_2.0_spec</artifactId>
-            <version>${version.jaxrs.api}</version>
+            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
+            <version>${version.jaxrs-api}</version>
             <scope>provided</scope>
         </dependency>
+-->
```

Comment:
This is commented out. Is it intentional?

Reply:
okay